app/src/main/java/com/nextcloud/talk/controllers/WebViewLoginController.java

```diff
@@ -169,6 +175,8 @@ protected void onViewBound(@NonNull View view) {
         webView.clearHistory();
         WebView.clearClientCertPreferences(null);
 
+        webViewFidoBridge = WebViewFidoBridge.createInstanceForWebView((AppCompatActivity) getActivity(), webView);
```

Comment:
Maybe we should do a check if getActivity() != null?

Reply:
Hm, if getActivity() is null a lot of other lines would also fail (such as the next line with ``CookieSyncManager.createInstance(getActivity());``).
What about an early exit for the whole ``onViewBound()`` method if getActivity is null?

Reply:
I don't disagree with you. There's a need to rewrite this whole thing to make it better. So it's probably fine for now :)